src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -405,6 +423,28 @@ struct VSOutputs {
       }
 
       case 'unorm': {
+        if (formatInfo.bytesPerComponent === 'packed') {
+          assert(format === 'rgb10a2'); // This is the only packed format for now.
+          assert(bitSize === 0);
+
+          /* prettier-ignore */
+          const data = [
+            [0, 0, 0, 0],
+            [1023, 1023, 1023, 3],
+            [243, 567, 765, 2],
+          ];
+          const vertexData = new Uint16Array(data.map(makeRgb10a2)).buffer;
+          const expectedData = new Uint16Array(data.map(normalizeRgb10a2).flat()).buffer;
+
+          return {
+            shaderBaseType: 'f32',
+            testComponentCount: data.length,
+            expectedData,
+            vertexData,
+            floatTolerance: 0.1, // TODO: Change?
```

Comment:
Yes, this should be very small, like 1 / 1023 at most

Reply:
Agreed.